monash-human-power/raspicam
# Update MQTT topics

## Description

This PR updates raspicam to use the modified [`topics.py` from the common repo](https://github.com/monash-human-power/common/blob/hallgchris/mqtt-refactor/mhp/topics/topics.py). This mainly amounts to renaming some things and using some fancy syntax for comparing and building MQTT topics.

I've also sneaked in a fix for CICD, which had broken :(

This PR depends on monash-human-power/common#12. Once that PR is merged, I'll need to update `pyproject.toml`. Don't approve before this, otherwise I'll totally forget haha (pls still review though)

## Screenshots

n/a, same as before

## Steps to Test

Check that `overlay_new.py` can display V2 and V3 data, V2 BOOST data, and messages. Check that it and `orchestrator.py` publishes all the correct status messages.

Keep in mind that if you're testing them with other projects, you'll need to use the updated PRs for them too so that they MQTT topics match!
